__tests__/app.test.js

```diff
@@ -3,7 +3,7 @@ const supertest = require('supertest')
 const seed = require('../db/seeds/seed')
 const db = require('../db/connection')
 const testData = require('../db/data/test-data/index')
-const { forEach } = require('../db/data/test-data/articles')
+require('jest-sorted')
```

Comment:
we shouldnt need to require in jest-sorted here, it should be added in the package.json as stated in the docs

Jest >v24
Add jest-sorted to your Jest setupFilesAfterEnv configuration. See for help

For example, add the following to your package.json at the root level. See configuring jest for more info.

"jest": {
  "setupFilesAfterEnv": ["jest-sorted"]
} 

add this at root level, remove the require and it should still work